python/res/enkf/jobs/enkf_scaling_get_data.py

```diff
@@ -0,0 +1,173 @@
+from copy import deepcopy
+
+import pandas as pd
+
+from res.enkf.export import GenDataCollector, SummaryCollector, SummaryObservationCollector
+
+
+def get_measured_data(ert, events):
+    """
+    Main entry point, will read simulation and observed data,
+    filter on user specified index, and filter out outliers.
+    """
+
+    measured_data = _get_data(ert, events.keys)
```

Comment:
This looks strange. It is ok to store intermediary results in separate descriptive variables